src/gentropy/datasource/open_targets/l2g_gold_standard.py

```diff
@@ -51,7 +52,9 @@ def parse_positive_curation(
             )
             .withColumn(
                 "studyLocusId",
-                StudyLocus.assign_study_locus_id(f.col("studyId"), f.col("variantId")),
+                StudyLocus.assign_study_locus_id(
+                    f.col("studyId"), f.col("variantId"), f.col("finemappingMethod")
```

Comment:
The `finemappingMethod` column will be missing from the dataframe. check `select` above.